newswires/client/src/App.tsx

```diff
@@ -262,35 +324,23 @@ export function App() {
 					{status !== 'error' && (
 						<>
 							<EuiShowFor sizes={['xs', 's']}>
-								{view === 'item' ? <ItemData id={selectedItemId} /> : <Feed />}
+								{view === 'item' &&
+									(isPoppedOut ? (
+										<ResizableContainer
+											Feed={<Feed />}
+											Item={<ItemData id={selectedItemId} />}
+										/>
+									) : (
+										<ItemData id={selectedItemId} />
```

Comment:
for now keeping the pattern of opening items on their own for small screens, but allowing split view if it's a small screen and open in ticker mode